libs/designer/src/lib/ui/connections/dropzone.tsx

```diff
@@ -135,11 +140,19 @@ export const DropZone: React.FC<DropZoneProps> = ({ graphId, parentId, childId,
             >
               <FocusZone>
                 <div className="msla-add-context-menu">
-                  <ActionButton iconProps={{ imageProps: { src: AddNodeIcon } }} onClick={openAddNodePanel}>
+                  <ActionButton
+                    iconProps={{ imageProps: { src: AddNodeIcon } }}
+                    onClick={openAddNodePanel}
+                    data-automation-id={`msla-add-action-${parentId}-${childId}`.replace(/\W/g, '-')}
```

Comment:
Missing utility function in this file

Reply:
I didn't use the utility function because I wanted to stick with the pattern established by the 'buttonId' field above.